config/routes.rb

```diff
@@ -1,28 +1,30 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
```

Comment:
I would leave this in there because someone in the future might need this info and it comes with the file.